Sources/XcodeGenKit/SourceGenerator.swift

```diff
@@ -119,7 +119,7 @@ class SourceGenerator {
                 let models = (try? path.children()) ?? []
                 let modelFileReference = models
                     .filter { $0.extension == "xcdatamodel" }
-                    .sorted()
+                    .sorted { $0.string.localizedStandardCompare($1.string) == .orderedDescending }
```

Comment:
Even though this will be rewritten in `sortGroups` shouldn't this be `.orderedAscending`?

Reply:
Then I've to do something like ` currentVersion: modelFileReference.last?.reference` over here https://github.com/rohitpal440/XcodeGen/blob/4d964c07861cd524801b85d4edf611c77e1d2c3b/Sources/XcodeGenKit/SourceGenerator.swift#L134
 to make higher version of `.xcdatamodel` file as current

Reply:
So what do you suggest me to do? 
1. Keep the code as in this PR
OR
2. Compare using `.orderedAscending` and update the code for getting currentVersion to `currentVersion: modelFileReference.last?.reference`

In both ways I'm going to get the same result, chose approach 1 since the change stays in one place instead of 2.

Reply:
I'd go with option 2. As you want to sort in ascending order